### DISCUSSION TITLE
[Feature]: Wildcard Domains for each Project

### DISCUSSION THREAD

**Discussion by peaklabs-dev** [coollabsio/coolify#3140] [Feature Requests]
### Description

let us add wildcard domains to each project so not only the server level wildcard but project level wildcard domain for example *.domain.com and for the dev environment *.domain.dev
cc @DeVoresyah

### Minimal Reproduction (if possible, example repository)

-

### Version

-

**Comment by pksorensen** [comment]
@ayntk-ai  - does this mean its not possible currently to do so? 

I need it on a project and was assuming it was just to set up the traefik labels?

**Comment by peaklabs-dev** [comment]
@pksorensen At the moment you can not add wildcard domains over the UI for projects no (which would be ideal). Also their needs to be logic if there is a Project level wildcard domain and a server level wildcard domain for wiche to choose. And no traffic labels are not used for wildcard domains just the FQDN of the application.

**Comment by geovanygameros** [comment]
I'm also trying to set up a wildcard domain for a specific application, but without success so far. I'm following [this guide](https://coolify.io/docs/knowledge-base/traefik/wildcard-certificates). Have any of you found a solution?

**Comment by priard** [comment]
Hi everyone,

Is there any update on this? The ability to set a wildcard domain per project seems like a sufficient solution for now, as there's currently no easy or convenient way to configure it. 

In my current Coolify Cloud setup, which spans several servers, I host a few large websites that generate tens of millions of views. Unfortunately, traffic to outdated subdomains (which may have built up over the years) isn’t being properly handled due to the lack of wildcard support. At the same time, I can’t simply redirect all traffic to the root, as some subdomains need to function correctly.

@peaklabs-dev Your proposal for configuring wildcard domains on a per-project basis seems very sensible. Are there any plans to implement this feature soon?

**Comment by DeVoresyah** [comment]
let me put bounty on this

**Comment by rihards-simanovics** [comment]
I hit the wall with this one today. Kind of need this for some semi-complex applications. Since Coolify is stable now, is it possible that this will be added in a minor release, some time this/next year?

**Reply by rihards-simanovics** [reply]
Managed to get it working with labels, so I guess there is that as a workaround.
